lib/api-extensions.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const ApiGroup = require('./api-group');
+const CustomResourceDefiniton = require('./custom-resource-definition');
+
+class ApiExtensions extends ApiGroup {
+  constructor(options) {
+    options = Object.assign({}, options, {
+      path: 'apis/apiextensions.k8s.io',
+      version: options.version || 'v1beta1',
+      groupResources: [
+        'customresourcedefinition'
```

Comment:
pluralize: `customresourcedefinitions`

The code automatically creates a singular alias so `apiExtensions.customresourcedefinitions` and `apiExtensions.customresourcedefinition` will both work, but you need to specify the plural version here.